manifests/group.pp

```diff
@@ -36,6 +36,14 @@
         unless  => "yum grouplist hidden '${name}' | egrep -i '^Installed.+Groups:$'",
         timeout => $timeout,
       }
+      if $ensure == 'latest' {
```

Comment:
An acceptance test would make this easier to merge.

Reply:
In order to do an acceptance test I will have to add a custom package group, install it, then modify that package group to add more packages and then re-run Puppet to ensure the package group is fully installed with the newly added packages.  I can do that but will add a lot of complexity to the acceptance tests.

The way I use this feature is managing a local base package group to avoid dozens of `Package` resources in Puppet on fresh installs which is slow. I sometimes update that package group and this pull request ensures that `ensure => latest` keeps the group fully installed. I don't think it can be tested with built-in package groups as it requires packages being added to the group after initial install of the group.